python/lsst/meas/extensions/scarlet/deblend.py

```diff
@@ -112,21 +108,31 @@ def _computePsfImage(self, position=None):
     return psfImage
 
 
-def deblend(mExposure, footprint, log, config):
+def deblend(mExposure, footprint, config):
+    """Deblend a parent footprint
+
+    Parameters
+    ----------
+    mExposure: `MultibandExposure`
```

Comment:
See https://developer.lsst.io/python/numpydoc.html#py-docstring-parameters
- Must have a space before and after the colon
- Parameter types should have the full namespace (`lsst.foo.bar`); I haven't checked to see if they work without them
- It would be nice to maintain consistency (always end with a period; only add hyphens for lists)